src/main/java/io/cdap/plugin/gcp/bigquery/sink/AbstractBigQuerySinkConfig.java

```diff
@@ -69,6 +87,21 @@ public boolean isAllowSchemaRelaxation() {
     return allowSchemaRelaxation;
   }
 
+  @Nullable
+  public Boolean isCreatePartitionedTable() {
```

Comment:
nit: rename to shouldCreatePartitionedTable

Reply:
I think it's better if this getter returns a boolean so that callers don't need to check for null. If the value is null, it should return false.

Reply:
Done.